content/app-dev/create-provider/_index.en.md

```diff
@@ -5,10 +5,21 @@ weight: 5
 draft: false
 ---
 
-Creating a capability provider is one of the more involved tasks when working within the wasmCloud ecosystem. First, you need to decide if you're creating a new provider for an existing contract or if you're creating an entirely new contract.
+Creating a capability provider is one of the more involved tasks when working within the wasmCloud ecosystem. First, you need to decide if you're creating a new provider for an existing contract or if you're creating an entirely new contract. If you don't need to create a new contract, then you can skip to the second step in this guide.
 
-If you do not need to create a new contract, then you can skip to step 2 in this guide:
+### The Sample Use Case
 
-1. [Create a new contract](./new_contract)
-1. [Create the Rust capability provider](./rust)
-1. [Consume the new capability provider](./consuming) (create actors that use this provider)
+For the purposes of this guide, we're going to be creating a capability provider for a _payments service_ that accepts payments from customers. When composing applications with actors and capabilities, actors in an ecommerce setting may contain business logic that includes the need to accept payments from customers. There are a couple of ways we could solve this problem:

```

Comment:
For the purposes of this guide, we're going to create a capability provider for a _payment service_ that accepts payments from customers.